addons/io/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/api/HueDevice.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.io.hueemulation.internal.api;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Hue API device object
+ * 
+ * @author Dan Cunningham
+ *
+ */
+public class HueDevice {
+    public HueState state;
+    public String type = "Extended color light";
+    public String name;
+    public String modelid = "LCT001";
```

Comment:
For the sake of best emulation, it is probably cleaner to expose the lights as "Dimmable Light" only, see http://www.developers.meethue.com/documentation/supported-lights, i.e. the type should be "Dimmable Light" and the modelid e.g. "LWB004".